tests/BlazingSQLTest/Runner/runTest.py

```diff
@@ -322,22 +325,70 @@ def print_fixed_log(
     logger.info(total_time)
 
 
-def print_query_results(
+def comparing_results(  pdf1,
```

Comment:
this `comparing_results` and the above `compare_results` have very very similar names and it can be confusing. I propose we call this one `compare_test_results` and the above `compare_result_values` since this one is about all the results while the one above is only looking at the actual values